rr/config/.rr.yaml

```diff
@@ -0,0 +1,20 @@
+http:
+  address: :8080
```

Comment:
should we use `$PORT` here? I think by default fpm runs on `$PORT`

Reply:
Thanks, fixed